descwl_coadd/coadd.py

```diff
@@ -443,35 +495,30 @@ def make_stacker(mask, coadd_dims):
     )
 
 
-def get_coadd_stats_control(mask):
+def get_coadd_stats_control():
     """
     get a afw_math.StatisticsControl with "and mask" set
 
-    Parameters
-    ----------
-    mask: mask for setAndMask
-        Bits for which the pixels will not be added to the coadd.
-        e.g. we would not let EDGE pixels get coadded
-
     Returns
     -------
     afw_math.StatisticsControl
     """
+
+    mask = afw_image.Mask.getPlaneBitMask(FLAGS2CHECK_FOR_COADD)
+
     stats_ctrl = afw_math.StatisticsControl()
     stats_ctrl.setAndMask(mask)
     # not used by the Accumulator
     # stats_ctrl.setWeighted(True)
     stats_ctrl.setCalcErrorFromInputVariance(True)
 
-    # TODO when we make the BRIGHT plane, we will have BRIGHT at 0.0 here but
-    # not so for others (e.g. SAT should be 0.1 or whatever)
-    # TODO make this part of a configuration
+    # the mask here is going to be just EDGE and we always
+    # want to watch for it. it is a bug if EDGE is included
+    # for regular images (not psf)
+    mask_prop_thresh = {}
+    for flagname in FLAGS2CHECK_FOR_COADD:
+        mask_prop_thresh[flagname] = 0.001
```

Comment:
Why the magic number 0.001 vs 0?

Reply:
depends on if the test is `>` or `>=`

Reply:
just checked, the test is `>` so we need some small finite number, 0.001 seems ok

Reply:
ugh sorry had it backwards, that means zero is fine

Reply:
I am confused. Can you explain in more detail?

Reply:
Ahhh nvm.

Reply:
no problem, was in a hurry as I need to get on the road